core/primitives/src/utils.rs

```diff
@@ -470,7 +470,7 @@ impl fmt::Debug for dyn CompiledContractCache {
 /// objects using tracing.
 ///
 /// tracing::debug!(target: "diagnostic", value=%ser(&object));
-pub fn ser<'a, T>(object: &'a T) -> Serializable<'a, T>
+pub fn ser<T>(object: &T) -> Serializable<T>
```

Comment:
It's a bit weird that clippy suggests that. I *think* the idiomatic way to write these nowdays is 

```rust
pub fn ser<T>(object: &T) -> Serializable<'_, T>
```

That is, you do want to indicate to the reader that there's a lifitime in the struct.